### DISCUSSION TITLE
那既然需要故意触发的问题会被删除，那岂不是安全漏洞、其他人复现都可以被作者以需要故意触发的问题的理由删除？

### DISCUSSION THREAD

**Discussion by UserNameiiiii** [Meloong-Git/PCL#3012] [提问] [answered]
@LTCatt 我觉得那既然需要故意触发的问题会被删除，那岂不是安全漏洞、其他人复现都可以被作者以需要故意触发的问题的理由删除？不都是需要故意触发的问题吗？

**Comment by MoYuan-CN** [comment, marked as answer]
需要故意触发的问题不会受理，安全问题不会公开讨论
另在讨论的时候请保持一个平和的心态，有助于解决问题哦~

**Reply by UserNameiiiii** [reply]
那我知道了不会受理会被删除，那为啥千万别点一直都在，不是也是不会受理吗？

**Reply by MoYuan-CN** [reply]
只是个彩蛋。
千万别点不被认作为 Bug

**Reply by UserNameiiiii** [reply]
那为什么我用我的脚写出来的C++程序也能将PCL2、BakaXL搞报错？

**Reply by MoYuan-CN** [reply]
这我不太清楚，你的程序可能有侵入性，导致启动器运行时产生了意外的行为

**Reply by UserNameiiiii** [reply]
我就给PCL2发了一张通知信也就是消息，又不是通知关闭、销毁、退出窗口循环，为什么能报错呢，程序&源代码：
[程序.zip](https://github.com/Hex-Dragon/PCL2/files/13702107/default.zip)

**Reply by burningtnt** [reply]
0. 你的程序完全不符合 Windnows 消息系统的规范，导致 VB 框架崩溃也是理所当然
1. 即使你认为 PCL2 应该阻止此问题，但其实，这个崩溃是由于 VB 框架没有处理不合法的消息格式导致的，PCL2 无法修复
2. 你的程序**故意向 PCL2 发送不合法的消息**这个行为已经属于“故意触发的范畴”

**Reply by MoYuan-CN** [reply]
Wind "n" ows
~~（雾）~~

**Reply by iPodtouch6557** [reply]
不要找chhhhhhhhhha
这里指的是不常遇到 几乎没有人会这么干的问题 以及千万别点，不要找茬

**Reply by UserNameiiiii** [reply]
这个是什么:ḁ̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̥̊̊̊̊̊̊̊̊̊̊̊̊̊̊̊̊̊
